src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml

```diff
@@ -37,9 +37,15 @@
       Command="{x:Static nuget:Commands.FocusOnSearchBox}"
       Gesture="CTRL+L" />
   </UserControl.InputBindings>
-  <DockPanel
-    x:Name="_root"
-    LastChildFill="True" >
+
+  <ScrollViewer
+    VerticalScrollBarVisibility="Disabled"
+    HorizontalScrollBarVisibility="Auto">
+    <DockPanel
+      x:Name="_root"
+      LastChildFill="True"
+      Width="{Binding ActualWidth, ElementName=_self}"
+      MinWidth="634">
```

Comment:
where does this come from? also make sure you test this on high DPI

Reply:
634 is the sum of the min widths of the grid columns (300 for the first, 4 for the seconds and 330 for the third). I could make them all constants in code and bind them. I did not invent this limits, I just constrained it to what it was previously.

Reply:
@rohit21agrawal I have tested it on high DPI and it works as expected. This change gives a better usability to high DPI scenarios where the whole UI was not shown before.

Reply:
a comment to explain 634 would be nice to have.